docs/requirements.txt

```diff
@@ -0,0 +1,58 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
+# To update, run:
+#
+#    pip-compile docs/requirements.in
+#
+alabaster==0.7.12
+    # via sphinx
+babel==2.10.3
+    # via sphinx
+certifi==2022.6.15
+    # via requests
```

Comment:
da's een oude versie :) mijn advies zou zijn om even deze dependencies netjes op orde te krijgen en ik raad [`uv`](https://docs.astral.sh/uv/) als build-tool aan - als't goed is zou deze ook prima moeten werken op windows.

Mits `uv` geïnstalleerd is, dan wordt het (in de `docs` folder):

1. (Eenmalig) installeer Python versie. Ik neem 3.12 omdat dat de meest recente stable versie is - voor Sphinx maakt het niet zo heel erg veel uit (3.10 is over 2 jaar end of life)
    
    ```bash
    uv python install 3.12
    ```

2. (Eenmalig) maak een virtualenv (isoleren van dependencies).

    ```bash
    uv venv --python 3.12
    # creates .venv folder, add this to your .gitignore
    ```
    
3. (Eenmalig) resolve alle dependencies, met de volgende file contents:

    ```
    # requirements.in
    sphinx
    sphinx_rtd_theme
    ```

    ```bash
    uv pip compile --output-file requirements.txt requirements.in
    ```
    
    Dit zorgt voor de file requirements.txt met alle gepinde versies

4. Installeer dependencies
    
    ```bash
    uv pip install -r requirements.txt
    ```

5. (Eenmalig) bootstrap de documentatie configuratie:
    
    ```bash
    uv run sphinx-quickstart
    ```
    
    Dit zorgt voor de `Makefile` etc., en de `conf.py` file. Die kan je achteraf nog bewerken. Met name de theming is belangrijk:
    ```py
    # -- Options for HTML output -------------------------------------------------
    # The theme to use for HTML and HTML Help pages.  See the documentation for
    # a list of builtin themes.
    #
    html_logo = "logo.png"
    html_theme = "sphinx_rtd_theme"
    
    # Add any paths that contain custom static files (such as style sheets) here,
    # relative to this directory. They are copied after the builtin static files,
    # so a file named "default.css" will overwrite the builtin "default.css".
    html_static_path = ["_static"]
    html_css_files = [
        "theme_overrides.css",  # override wide tables with word wrap by Maykin
    ]
    ```
6. Build HTML documentation:
    
    ```bash
    uv run make html
    ```

Optioneel kan je ook [sphinx-autobuild](https://pypi.org/project/sphinx-autobuild/) lokaal gebruiken voor hot-reloading terwijl je docs schrijft.

Reply:
Heel fijn, dank voor de feedback! Ik heb de stappen uitgevoerd: b910a0e20421a01324e850b672b6f54a0b8bfea3